src/Riok.Mapperly/Descriptors/MappingBuilders/ImplicitCastMappingBuilder.cs

```diff
@@ -16,8 +16,6 @@ public static class ImplicitCastMappingBuilder
             return null;
 
         var conversion = ctx.Compilation.ClassifyConversion(ctx.Source, ctx.Target);
-        return conversion.IsImplicit
-            ? new CastMapping(ctx.Source, ctx.Target)
-            : null;
+        return conversion.IsImplicit ? new CastMapping(ctx.Source, ctx.Target) : null;
```

Comment:
Is there a setting to always break ternary operations into multiple lines? Personally, I find it easier to read

Reply:
There is no such setting. Csharpier does not have a lot of options on how to format source code (almost none expect for the line width and whether to use tabs). Although I don't agree with all formatting decisions I think to have one enforced standard for such a project is still beneficial.